CMakeLists.txt

```diff
@@ -1,17 +1,7 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.10...3.31)
 # Note: this is a header only library. If you have an older CMake than 3.5,
 # just add the CLI11/include directory and that's all you need to do.
 
-# Make sure users don't get warnings on a tested (3.5 to 3.30) version
-# of CMake. For most of the policies, the new version is better (hence the change).
-# We don't use the 3.5...3.30 syntax because of a bug in an older MSVC's
-# built-in and modified CMake 3.11
```

Comment:
Do we care about this bug anymore?

Reply:
I don't think so. Not sure we even support this old version of MSVC anymore.